src/SearchButton.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const SearchButton = () => {
+  return (
+    <div>
+      <button className="btn btn-primary">Search</button>
```

Comment:
I _think_ this should also have `type="submit"`, because it's intended to submit the enclosing form.

Not essential because for some reason the code still works without it- I think React is doing something clever here to make it work as you'd expect.